go/vt/sqlparser/ast_funcs.go

```diff
@@ -1195,8 +1195,25 @@ func compliantName(in string) string {
 	return buf.String()
 }
 
-func (node *Select) AddSelectExprs(selectExprs SelectExprs) {
-	node.SelectExprs = append(node.SelectExprs, selectExprs...)
+func (node *Select) AddSelectExprs(selectExprs *SelectExprs) {
+	if node.SelectExprs == nil {
+		node.SelectExprs = &SelectExprs{}
+	}
+	node.SelectExprs.Exprs = append(node.SelectExprs.Exprs, selectExprs.Exprs...)
+}
+
+func (node *Select) AddSelectExpr(expr SelectExpr) {
+	if node.SelectExprs == nil {
+		node.SelectExprs = &SelectExprs{}
+	}
+	node.SelectExprs.Exprs = append(node.SelectExprs.Exprs, expr)
+}
+
+func (node *Select) SetSelectExprs(exprs ...SelectExpr) {
+	if node.SelectExprs == nil {
+		node.SelectExprs = &SelectExprs{Exprs: exprs}
+	}
```

Comment:
nit to match the pattern used in the two functions above 

```suggestion
	if node.SelectExprs == nil {
		node.SelectExprs = &SelectExprs{}
	}
```